test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/ProjectDirectory.cs

```diff
@@ -103,6 +103,7 @@ void CreateDirectoryProps(string originalProjectRoot, string binariesRoot, strin
     <OriginalProjectRoot>{originalProjectRoot}</OriginalProjectRoot>
     <BinariesRoot>{binariesRoot}</BinariesRoot>
     <_RazorMSBuildRoot>$(OriginalProjectRoot)\..\..\..\src\Microsoft.AspNetCore.Razor.Design\bin\{configuration}\netstandard2.0\</_RazorMSBuildRoot>
+    <_RazorBuildServerPipeName>{BuildServerTestFixture.PipeName}</_RazorBuildServerPipeName>
```

Comment:
Think about just passing this in the command line args to MSBuild. That's much more first class and simple than this.